cel/cel_test.go

```diff
@@ -90,6 +90,14 @@ func Example() {
 
 	fmt.Println(out)
 	// Output:Hello world! Nice to meet you, I'm CEL.
+
+	// Calculates the estimate cost for the program.
+	min, max := EstimateCost(prg)
```

Comment:
Since this is an example, maybe don't include the cost test here. I'd be happy to have another cost-specific test.

Reply:
moved to a separate test